simulator/simulator_programming.md

```diff
@@ -73,53 +73,58 @@ Your robot has one servo board attached, the jaws of the robot are controlled by
 |-------|-----------|
 | 0     | Left Jaw  |
 | 1     | Right Jaw |
+| 2     | Lifter    |
 
-Setting each servo to -1 fully opens the respective jaw.
+Setting each servo to -1 fully opens the respective jaw, setting them to 1 fully opens them.
+
+Setting the lifter to -1 fully lowers the lifter, setting it to 1 fully raises it.
 
 The servo board has the part code `srXYZ2`, but since only a single servo board is attached it can be referenced as `R.servo_board`.
 
-### Ruggeduino
+### Arduino
 
-Your robot has a microswitch and six distance sensors, attached to the digital and analogue pins respectively. These are all attached to a single ruggeduino.
+Your robot has a microswitch and six distance sensors, attached to the digital and analog pins respectively. These are all attached to a single arduino.
 
-Because these sensors are pre-attached to the ruggeduino, you do not need to set its `pin_mode`.
+Make sure you have set the correct [pin_mode]({{ site.baseurl }}/programming/arduino/sr_firmware#setting-pin-modes), depending on what device you're using.
 
 #### Microswitches
 
+The rear of the robot has a wide microswitch.
+
 The microswitch is attached to digital pin 2:
 
-| Pin | Location |
-|-----|----------|
-| 2   | Back     |
+| Pin | Location | Required Mode |
+|-----|----------|---------------|
+| 2   | Back     | `INPUT`       |
 
 This is shown as a red coloured block on the robot. Using the `digital_read`  method, you'll receive a `bool` telling you whether the switch is currently actuated.
 
 #### Distance Sensors
 
-Analogous to ultrasound sensors, distance sensors allow you to retrieve the distance between your robot and an object. These are attached to analogue pins A0-A5:
+Analogous to ultrasound sensors, distance sensors allow you to retrieve the distance between your robot and an object. These are attached to analog pins A0-A5:
 
-| Pin | Location |
-|-----|----------|
-| A0  | Front Left |
+| Pin | Location    |
```

Comment:
Should we have the required pin modes here too? (Presumably also `INPUT`?)

Reply:
I thought they didn't care what mode they were in?

Reply:
turns out they need to be in `INPUT` at least